midi_app_controller/gui/utils.py

```diff
@@ -141,8 +146,14 @@ def get_selected_action_id(self) -> Optional[str]:
 
 def is_subpath(path: Path, subpath: Path) -> bool:
     """Checks if one path represents a file/directory inside the other directory."""
-    path_str = str(subpath.resolve().absolute())
-    subpath_str = str(path.resolve().absolute())
+    if subpath is not None:
+        path_str = str(subpath.resolve().absolute())
+    else:
+        path_str = ""
+    if path is not None:
+        subpath_str = str(path.resolve().absolute())
+    else:
+        subpath_str = ""
```

Comment:
I don't think this is a good idea. This way `is_subpath(None, "a/b")` is True and `is_subpath(None, None)` is True, and `is_subpath("a", None)` is False, and it isn't clear to me what any of this means. I think checking for None outside the function makes more sense. Else, there should be more description of what this function does with Nones, and also type hints should be changed to `Optional[None]`.